tools/gulp/tasks/e2e.ts

```diff
@@ -7,11 +7,14 @@ import {
   vendorTask, sequenceTask, serverTask
 } from '../task_helpers';
 
+const gulpRunSequence = require('run-sequence');
 
 const appDir = path.join(SOURCE_ROOT, 'e2e-app');
 const outDir = DIST_ROOT;
 const PROTRACTOR_CONFIG_PATH = path.join(PROJECT_ROOT, 'test/protractor.conf.js');
 
+/** Method to stop a running e2e web server, which may have not exited properly */
+let stopE2eServer: () => void = null;
```

Comment:
Sounds good. I had the `cleanupTasks` method because I thought that it would be good in the future to just have all cleanups at one place.